hack/label_generate/process_label.py

```diff
@@ -0,0 +1,110 @@
+import os
+
+import yaml
+import requests
+
+label_dir = os.path.dirname(os.path.abspath(__file__)) + '/../../label_sync/'
+
+# old label -> new label mapping
+label_mapping = {
+    # 'approved': 'status/approved',
+    'area/os/centos': 'os/centos',
+    'area/os/macos': 'os/macos',
+    'area/os/ubuntu': 'os/ubuntu',
+    'area/os/windows': 'os/windows',
+    'area/platform/aws': 'platform/aws',
+    'area/platform/azure': 'platform/azure',
+    'area/platform/gke': 'platform/gcp',
+    'area/platform/minikube': 'platform/minikube',
+    'area/release-eng': 'area/build-release',
+    'area/releasing': 'area/build-release',
+    'area/testing': 'testing',
+    'area/ui': 'area/front-end',
+    'bug': 'problems/bug',
+    'cloud/azure': 'platform/azure',
+    'cloud/gke': 'platform/gcp',
+    'do-not-merge/work-in-progress': 'status/in progress',
+    'enhancement': 'improvement/enhancement',
+    'inference': 'area/inference',
+    # 'lgtm': 'status/lgtm',
+    'question': 'community/question'
+}
+
+
+class LabelColorMapping(object):
+    color_mapping_general = {
+        'area': 'd2b48c',
+        'os': '8cd2b4',
+        'addition': 'd28caa',
+        'community': 'ecfc15',
+        'platform': '2515fc',
+        'problems': 'fc2515',
+        'priority': {'p0': 'db1203', 'p1': 'db03cc', 'p2': 'fc9915'},
+        'status': {'approved': '03db12', 'backlog': 'bc9090', 'done': 'fca42e',
+                   'icebox': '808080', 'in progress': 'ffa500', 'lgtm': '2efca4'},
+        'default': '00daff'
+    }
+
+    @staticmethod
+    def get_color(label):
+        color_mapping = LabelColorMapping.color_mapping_general
+        names = '/'.split(label)
+        if names[0] not in color_mapping:
+            return color_mapping['default']
+        if isinstance(color_mapping[names[0]], dict):
+            return color_mapping[names[0]][names[1]]
+        return color_mapping[names[0]]
+
+
+# generate label config in yaml
+# Two inputs:
+#   1. csv file exported from community label definition
+#   2. old label -> new label mapping
+def csv_to_yml():
+    input_csv = os.path.dirname(os.path.abspath(__file__)) + "/kubeflow_label_standard.csv"
+    output_f = label_dir + "kubeflow_label.yml"
+    name_to_label = {}
+    with open(input_csv) as raw_csv:
+        pref = ''
+        label_list = []
+        for line in raw_csv:
+            words = line.strip(' \t\n\r').split(',')
```

Comment:
Why not use the `csv` library to parse instead of manually doing all the heavylifting